certified-connectors/Gravity Forms Professional/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1657 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Gravity Forms Professional",
+    "description": "Connect your WordPress forms powered by Gravity Forms to the Power Platform. This is the Professional and Enterprise version, offering greater features.",
+    "contact": {
+      "name": "Reenhanced",
+      "url": "https://www.reenhanced.com",
+      "email": "support@reenhanced.com"
+    }
+  },
+  "host": "we.buildbettersoftware.com",
+  "basePath": "/gfpa",
+  "schemes": ["https"],
+  "consumes": [ "application/json" ],
+  "produces": [ "application/json" ],
+  "paths": {
+    "/resources": {
+      "get": {
+        "tags": ["Resources"],
+        "summary": "Get resources",
+        "description": "Get resources",
+        "x-ms-visibility": "internal",
+        "operationId": "GetResources",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Resource"
+              }
+            }
+          },
+          "403": {
+            "description": "Unauthorized"
+          },
+          "429": {
+            "description": "Throttled"
+          },
+          "500": {
+            "description": "Server Error"
+          }
+        }
+      }
+    },
+    "/resources/forms/1": {
+      "options": {
+        "operationId": "FormSchema",
+        "summary": "Retrieve schema for the form",
+        "description": "This is where the magic happens for member objects",
+        "x-ms-visibility": "internal",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "get": {
+                  "type": "object"
+                },
+                "post": {
+                  "type": "object"
+                },
+                "schema": {
+                  "type": "object"
+                },
+                "delete": {
+                  "type": "object"
+                },
+                "patch": {
+                  "type": "object"
+                }
+              }
+            }
+
+          }
+        }
+      }
+    },
+    "/resources/forms": {
+      "get": {
+        "operationId": "GetForms",
+        "tags": ["Forms"],
+        "summary": "Get forms",
+        "description": "Get a list of Gravity Forms forms",
```

Comment:
Is this repeated word? if so, can you please fix it?

Reply:
Hi, the product name is "Gravity Forms", and it has multiple different entities under it, the "form" being one of the entities.

This is correct, but if you like we can just make this read "Get a list of forms". Will update.